pdk_regression.yml

```diff
@@ -0,0 +1,10 @@
+name: pdk_regression
+channels:
+  - litex-hub
+  - defaults
+dependencies:
+  - klayout
+  - python
+  - pip
```

Comment:
@proppy I followed the example: https://github.com/SymbiFlow/make-env/blob/33b80bd32c30fb8affd0fd5cda544d1bca075593/test/environment.yml#L14

Are you sure it will work if I removed it?

Reply:
ah my bad: you're right, that's what the conda docs have too:
https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-file-manually